src/extra/treemacs-evil.el

```diff
@@ -85,7 +85,8 @@ Then call ORIG-FUN with its ARGS and reopen treemacs if it was open before."
 (evil-define-key 'treemacs treemacs-mode-map (kbd "yy")     #'treemacs-copy-path-at-point)
 (evil-define-key 'treemacs treemacs-mode-map (kbd "yf")     #'treemacs-copy-file)
 (evil-define-key 'treemacs treemacs-mode-map (kbd "gr")     #'treemacs-refresh)
-(evil-define-key 'treemacs treemacs-mode-map [down-mouse-1] #'ignore)
```

Comment:
You should mention drag support in the readme under Detailed Feature List / Mouse Interface.